schemas/telemetry/frecency-update/frecency-update.1.schema.json

```diff
@@ -0,0 +1,59 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "properties": {
+    "client_id": {
```

Comment:
This is sent as `clientId` generally. You should use the template value `@TELEMETRY_CLIENTID_1_JSON@`, which includes the name and pattern.